File: PLAYGROUND/CFUNC/prime.c
# include <stdio.h>
# include <stdbool.h>
#include <math.h>

int is_prime(int number)
{
    int sq = (int)sqrt(number);
    if (number<=1)
    {
        return false;
    }

    for (int i=2; i<=sq; i++)
    {
        if (number % i == 0)
        {
            return false;
        }
    }
    return true;
}

int main(void)
{
    int prime[1000];
    int range[2];
    int index = 0;
    printf("Enter range of numbers:");
    for (int i=0; i<2; i++)
    {
        scanf("%d",&range[i]);
    }
    for (int j=range[0]; j<=range[1]; j++)
    {
        if (is_prime(j))
        {
            prime[index]=j;
            index++;
        }
    }
    for (int k=0; k<index; k++)
    {
        printf("%d ,",prime[k]);
    }

    printf("\nThere are [%d] prime numbers between %d and %d",index,range[0],range[1]);

}

File: PLAYGROUND/prime.py
def is_prime(number):
    if number <= 1:
        return False
    for  i in range(2,int(number**0.5)+1):
        if number % i == 0:
            return False
    return True

ranges=input("Enter range:").split(",")
range_list=[int(i) for i in ranges]
prime=[]
count=0

for j in range(range_list[0],range_list[1]):
    if is_prime(j):
        prime.append(j)

for k in prime:
    count+=1   
print(prime)
print(f"There are [{count}] prime numbers between {range_list[0]} and {range_list[1]}")

